src/js/components/Pagination/Pagination.js

```diff
@@ -199,27 +204,69 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const paginationControls = (
+      <Nav
+        a11yTitle={ariaLabel || a11yTitle || 'Pagination Navigation'}
+        ref={ref}
+      >
+        <Box as="ul" {...theme.pagination.controls}>
+          {controls.map((control, index) => (
+            /* Using index as key (as opposed to a unique id) seems to
+             * help React prioritize rendering the updated controls as
+             * desired. Whereas, using a unique id resulted in rendering
+             * the active control with an undesired lag. */
+            // eslint-disable-next-line react/no-array-index-key
+            <PageControl key={index} size={size} {...control} />
+          ))}
+        </Box>
+      </Nav>
+    );
+
+    // for backwards compatibility
+    if (!summary && !stepOptions)
+      return (
+        <StyledPaginationContainer
+          flex={false}
+          {...theme.pagination.container}
+          {...rest}
+        >
+          {paginationControls}
+        </StyledPaginationContainer>
+      );
+
     return (
       <StyledPaginationContainer
+        direction="row"
+        align="center"
+        justify="between"
+        gap={{ column: 'small', row: 'xsmall' }}
+        wrap
         flex={false}
         {...theme.pagination.container}
         {...rest}
       >
-        <Nav
-          a11yTitle={ariaLabel || a11yTitle || 'Pagination Navigation'}
-          ref={ref}
-        >
-          <Box as="ul" {...theme.pagination.controls}>
-            {controls.map((control, index) => (
-              /* Using index as key (as opposed to a unique id) seems to
-               * help React prioritize rendering the updated controls as
-               * desired. Whereas, using a unique id resulted in rendering
-               * the active control with an undesired lag. */
-              // eslint-disable-next-line react/no-array-index-key
-              <PageControl key={index} size={size} {...control} />
-            ))}
+        {summary ? (
+          <PaginationSummary
+            messages={messages}
+            page={page}
+            step={step}
+            numberItems={total}
+          />
+        ) : undefined}
```

Comment:
Instead of a ternary with `undefined` does it make sense to just do something like this?
```suggestion
        {summary && (
          <PaginationSummary
            messages={messages}
            page={page}
            step={step}
            numberItems={total}
          />
        )}
```